plugin/tooling.py

```diff
@@ -535,3 +535,19 @@ def on_transport_close(self, exit_code: int, exception: Optional[Exception]) ->
         self._transport = None
         output = str(exception) if exception else '\n'.join(self._stderr_lines).rstrip()
         sublime.set_timeout(lambda: self._on_close(self._resolved_command, output, exit_code))
+
+
+class LspDoubleClickCommand(sublime_plugin.TextCommand):
```

Comment:
Are you on purpose not using `LspOnDoubleClickCommand` or you just haven't noticed that my suggestion has subtle difference (`*On*`)?